run_exotic_linux.sh

```diff
@@ -128,7 +128,18 @@ ${pip_runner} install --upgrade exotic
 echo "INFO: Launching EXOTIC user interface. ..."
 if ${pip_runner} freeze | grep -iq 'exotic' ;
 then
-    bash -c "exotic-gui"
+    if hash exotic-gui > /dev/null ;
```

Comment:
Instead of testing for `exotic-gui`, preference is to test for `~/.local/bin/exotic-gui` then add it to the PATH. 
Pseudocode: 
> if local-exotic-gui-in-bin ; then
>     export PATH=local-bin:${PATH}
> fi

Reply:
If you prefer, but I'll say why I don't think it is preferable:
- If exotic-gui is on the path, assuming that the ~/.local/bin version is preferable defeats PATH intention: it would cause that version to be run despite there being an intended version on the PATH
- Adding to the PATH has implications for any subprocesses or other commands executed later in the script - this is contrary to the intention of just dealing with the command in question.

On the point of the 'correct' way to get things right - the concern is the default environment as it exists on a major distribution, per default installation: ~/.local/bin is in .profile, but .profile is only applicable to a login shell, and x terminals aren't login shells, despite being the most common place to run a Tk app like exotic-gui.  Prescribing how to get the environment right kind of seems contradictory to what the ./run-exotic-linux.sh script does, which seems to be to assure that everything that is in place and needed to run is in fact in place - if that only works given some additional configuration modification, I'm not sure what purpose it serves.